mmap/src/main/c/Makefile

```diff
@@ -15,7 +15,7 @@ CFLAGS += -g -fPIC -O3 -Wall
 # These rely on lazy evaluation of each variable since LIBTYPE is not set until later.
 LIBNAME = libindeedmmap-$(ARCH).$(LIBTYPE)
 LIBSONAME = $(LIBNAME).1
```

Comment:
Why don't we update this version, too?

Reply:
We don't .. That's what becomes the 1.0.3 in the .so/.dylib name .. I only bumped the patch level since this is a very minor and internal change not a change to the external API..